torchaudio/_backend/utils.py

```diff
@@ -0,0 +1,292 @@
+from abc import ABC, abstractmethod
+from functools import lru_cache
+from typing import BinaryIO, Dict, Optional, Tuple, Union
+
+import torch
+from torchaudio.backend.common import AudioMetaData
+from torchaudio.io._compat import (
+    info_audio,
+    info_audio_fileobj,
+    load_audio,
+    load_audio_fileobj,
+    save_audio,
+    save_audio_fileobj,
+)
+from torchaudio._extension import _FFMPEG_INITIALIZED, _SOX_INITIALIZED
+from torchaudio.utils.sox_utils import get_buffer_size
+import torchaudio.backend.soundfile_backend as soundfile_backend
+
+
+class Backend(ABC):
+    @staticmethod
+    @abstractmethod
+    def info(uri: Union[BinaryIO, str], format: Optional[str]) -> AudioMetaData:
+        raise NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def load(
+        uri: Union[BinaryIO, str],
+        frame_offset: int = 0,
+        num_frames: int = -1,
+        normalize: bool = True,
```

Comment:
The name `normalize` caused confusion to many users who thought it was about volume normalization.
let's think about some alternative name for this argument.

Reply:
`normalize` seems fairly apt for the behavior the arg gates: normalizing the input int data to range [-1.0, 1.0] after converting to float32. Perhaps we can simply update the docstring to clarify this?